packages/components/src/UICompatibilityView/index.web.js

```diff
@@ -1,17 +0,0 @@
-// @flow
-import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-// TODO: we have `mobile-detect` in UIDevice
-// maybe use only one lib for
-import Bowser from 'bowser';
-
-import UICompatibilityViewCommon from './UICompatibilityViewCommon';
-
-export default class UICompatibilityView extends React.Component<*> {
-    render() {
-        if (!Bowser.msie) {
```

Comment:
@savelichalex FYI, this check was used to display a message for old versions of IE that we do not support their browser